docs/iris/src/whatsnew/contributions_1.9/docchange_2015-03-19_whats-new-aggregator.txt

```diff
@@ -0,0 +1,4 @@
+* Added a first pass of an aggregator to build a "What's New" document from submitted RST files for each change.
+
+ * Added a new page to the dev docs to explain the format for an entry.
+ * Added a reference to the new page, to the Dev Docs index.
```

Comment:
As above. This is a "working" example (i.e. it is likely to be aggregated into the 1.9 whatsnew) so is the template for all future aggregated whatsnew entries so should be _right_ - please shorten to one point, maximum two sentences.